### DISCUSSION TITLE
创建物化视图之后，部分正常分组查询执行失败

### DISCUSSION THREAD

**Discussion by JohnsonFromTianShan** [apache/doris#17272] [RFC]
在Doris-1.2.1上创建了一个多字段分组的物化视图后，在执行分组查询时，部分查询遇到“Field name is invalid”错误。
具体问题现象如下：
1. 表结构如图1所示：
![图1](https://user-images.githubusercontent.com/36793478/222036342-46e8ff4a-3e07-4d2f-b978-799e366812bb.png)
2. 未创建物化视图前，可正常执行date_id、store_id字段上的分组查询，如图2所示
![图2](https://user-images.githubusercontent.com/36793478/222037704-546dbb28-d4c2-4df4-a1b4-9fb4700f61a4.png)
3. 创建一个按date_id、store_id分组的物化视图，实现每日门店销售额预聚合，提高查询效率
![图3](https://user-images.githubusercontent.com/36793478/222036467-07b501af-d7c4-4f59-96a6-1672372cafca.png)
4. 再次执行按date_id、store_id字段分组查询，不但未能命中物化视图，反而报出“Field name is invalid”错误，见图4
5. 执行单个store_id字段分组查询，也报出“Field name is invalid”错误，见图4
![图4](https://user-images.githubusercontent.com/36793478/222036630-f09066fa-94b7-4b0f-a8ba-ff300191900a.png)
6. 也有执行成功的分组查询，如图5所示：按date_id、store_id+date_id分组查询，可以正常执行
![图5](https://user-images.githubusercontent.com/36793478/222036735-6ff4f7a6-c1ac-4752-b866-653c2c77ae92.png)

**Comment by LemonLiTree** [comment]


**Reply by JohnsonFromTianShan** [reply]
已加微，JohnsonFromTianShan是我。麻烦通过一下

**Comment by LemonLiTree** [comment]
fix
https://github.com/apache/doris/pull/17555
